src/Components/Notifications/MarkAllAsReadPanel.tsx

```diff
@@ -0,0 +1,61 @@
+import { Clickable, Flex, Text } from "@artsy/palette"
+import { useTranslation } from "react-i18next"
+import { useSystemContext } from "System"
+import createLogger from "Utils/logger"
+import { markAllNotificationsAsRead } from "./Mutations/markAllNotificationsAsRead"
+
+export const MARK_ALL_AS_READ_PANEL_HEIGHT = 40
+
+const logger = createLogger("MarkAllAsReadPanel")
+
+export interface MarkAllAsReadPanelProps {
+  unreadCounts: number
+}
+
+export const MarkAllAsReadPanel: React.FC<MarkAllAsReadPanelProps> = props => {
+  const { relayEnvironment } = useSystemContext()
+  const { t } = useTranslation()
+  const { unreadCounts } = props
```

Comment:
**nitpick(feel free to ignore):** what do you think of moving this to line 15 like ({ unreadCounts })

Reply:
done